components/i2c/spa06_003/definition.json

```diff
@@ -0,0 +1,10 @@
+{
+  "displayName": "SPA06-003",
+  "vendor": "Goertek",
```

Comment:
The datasheet calls it Goermicro, but the product page says Goertek so leaving this for now.
-Edit- It is Goertek as the umbrella brand, and Goermicro are one group of solutions

<img width="668" height="305" alt="Image" src="https://github.com/user-attachments/assets/6e640140-5c3a-4259-ac5e-e0c3d50ed737" />